images/trino/tests/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+data "oci_exec_test" "help" {
+  digest = var.digest
+  script = "${path.module}/container-test.sh"
+}
+
+resource "helm_release" "trino" {
+  repository       = "https://trinodb.github.io/charts/"
+  name             = "trino"
+  chart            = "trino"
+  namespace        = "trino"
```

Comment:
We should probably add some randomness here, if we ever want to be able to run multiples of these tests at the same time. If that's not possible for whatever reasons (some helm charts are bad neighbors), we can document that too.

Reply:
Agreed! Added randomness to the namespace.